spec/parsers/moov_parser_spec.rb

```diff
@@ -48,8 +48,13 @@ def deep_print_atoms(atoms, output, swimlanes = [])
 
       expect(result).not_to be_nil
       expect(result.nature).to eq(:video)
-      expect(result.width_px).to be > 0
-      expect(result.height_px).to be > 0
+      # This is a valid video recorded with QuickTime which its dimensions
+      # were not recognized.
+      # TODO: Investigate it why
+      unless mov_path.include?('Test_Meta_Atom_With_Size_Zero.mov')
```

Comment:
I don't think we should bypass something in a test like this but we need to figure out why the information does not get extracted correctly 🤔

Reply:
ok no problem, I will take a look on that!

Reply:
I created this https://github.com/WeTransfer/format_parser/pull/173 to tackle with this problem. After merging I will remove this IF, ok?

Reply:
I rebased this branch with #173 and I removed this conditional